### PR TITLE
Updates for PureScript 0.12

### PR DESCRIPTION
**Pull request by nsaunders**
* Updated imports
* Updated Bower dependencies
* Removed effect rows

### CODE REVIEW

**Review by paf31 — APPROVED** [review]
Great, thanks!

**Comment by paf31** [comment]
Could you please just update `purescript` and `pulp` versions in `package.json`? That should get Travis going again.

File: src/Node/Yargs/Applicative.purs [hunk 1]
```diff
@@ -49,9 +47,9 @@ instance applicativeY :: Applicative Y where
 -- | Compute some `Eff` action using command-line arguments, and run it.
 -- |
 -- | This function can throw exceptions.
-runY :: forall a eff. YargsSetup ->
-                      Y (Eff (exception :: EXCEPTION, console :: CONSOLE | eff) a) ->
-                         Eff (exception :: EXCEPTION, console :: CONSOLE | eff) a
+runY :: forall a. YargsSetup ->
```
Comment (paf31) [inline src/Node/Yargs/Applicative.purs]: This could probably all fit on a single line now.

**Comment by therealnicksaunders** [comment]
Indeed, good finds and thanks for the feedback @paf31!

**Comment by paf31** [comment]
Thanks!

**Pull request merged**